xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -378,10 +378,9 @@ void cancelEndpointDataWatch(String clusterName, EndpointWatcher watcher) {
 
   /**
    * Starts reporting client load stats to a remote server for the given cluster.
-   *
-   * @param loadStatsStore  a in-memory data store containing loads recorded by gRPC client.
    */
-  void reportClientStats(String clusterName, String serverUri, LoadStatsStore loadStatsStore) {
+  LoadReportClient reportClientStats(String clusterName, String serverUri) {
+    throw new UnsupportedOperationException();
```

Comment:
We can just do no-op as other methods in this class.

Reply:
We have to return something. Returning a `null` is not a good idea. This is same as no-op, but telling people not to call it unless they implemented it. But also tells them implementation is not required if they do not use this method.

The original thought for making `XdsClient` an abstract class with most methods having no-op implementation (instead of an `interface`) is that it allows a concrete subclass to only implement a subset of its features. For example, if we want to test its interactions with EDS only, we could just implement `watchEndpointData(...)` and `cancelEndpointDataWatch(...)` only. But seems we are not going to use in that way (subclass of `XdsClient` that implements partial features), we may want to make all methods in this class `abstract` (AKA, make `XdsClient` an interface).

Reply:
Oh I forgot it returns.